src/test/java/com/training/services/tests/VoyageCurrentSailValidateTest.java

```diff
@@ -43,4 +44,26 @@ public void testVoyageCurrentSailDateValidate(){
                 .matches("2020[0,1][0-2][0-3][0-9]")).
                 isTrue().as("is not in  the year 2020");
     }
+
```

Comment:
Lets change the name of the test class to some thing more generic.
VoyageCurrentSailValidateTest only makes sense for the first tc